src/components/Modals/CreateGnosisSafeModal/index.tsx

```diff
@@ -127,20 +150,54 @@ const CreateGnosisSafeModalContent: FunctionComponent<{
 						}}
 					/>
 					<label>Add Admins</label>
-					<ArrayInput
-						items={members}
-						onAdd={handleMemberAdd}
-						onRemove={handleMemberRemove}
-						placeholder="Paste address and press enter. Add more addresses if needed"
-						validator={(value: string) => (isAddress(value) ? null : "Bad address format")}
-					/>
+					<div className="create-gnosis-safe__row">
+						<div className="flexed">
```

Comment:
We're following BEM. `flexed` is rather something from atomic CSS. Please, adjust styling for ArrayInput, if you need, via direct overriding